src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -18,6 +18,9 @@
 
 namespace OmniSharp.Script
 {
+    using Dotnet.Script.NuGetMetadataResolver;    
+    using Microsoft.DotNet.InternalAbstractions;
```

Comment:
Please don't nest usings inside of namespaces. Put them with the rest of the namespaces at the top of the file.